### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by kingsleyzissou**

### CODE REVIEW

**Comment by kingsleyzissou** [comment]
@machi1990 @wtrocki 

Will this do, let me know if you have any feedback?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,96 @@
-# uniforms-ionic
-Uniforms ionic
+# Basic usage
```
Comment (wtrocki) [inline README.md:1]: Missing one sentence intro

File: README.md [hunk 2]
```diff
@@ -1,2 +1,96 @@
...
+To start using uniforms, we have to install three independent packages:
+
+1. Core
+2. Bridge
+3. Theme
+
+In this example, we will use the JSONSchema to describe our desired data format and style our form using Semantic UI theme.
+
+```shell
+npm install uniforms@3.0.0-alpha.4
```
Comment (wtrocki) [inline README.md]: 5 is latest relaae

File: README.md [hunk 3]
```diff
@@ -1,2 +1,96 @@
...
+import { AutoForm } from uniforms-semantic;
+
+import schema from './schema';
+
+export default function GuestForm() {
+  return <AutoForm schema={schema} onSubmit={console.log} />;
+}
+```
+
+And that's it! `AutoForm` will generate a complete form with labeled fields, errors list (if any) and a submit button.
```
Comment (wtrocki) [inline README.md:96]: I think we need to as follow up
Copy uniforms register here or.. include it into our packace

**Review by wtrocki — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kingsleyzissou) [reply README.md:1]: Cool, will add after planning

File: README.md (hunk 2 above)
Comment (kingsleyzissou) [reply README.md]: Maybe I can do `<latest>` or something?

File: README.md (hunk 3 above)
Comment (kingsleyzissou) [reply README.md:96]: Yeah I need to update the repo a little bit. Will try do it over the next couple of days. Need to sort out a ui issue with the patternfly repo too.

**Comment by wtrocki** [comment]
Amazing job!

**Comment by machi1990** [comment]
Thanks @kingsleyzissou

**Pull request merged**